totp/__init__.py

```diff
@@ -2,17 +2,48 @@
 #
 # Print a TOTP token getting the shared key from pass(1).
 
-import getpass
 import os
 import platform
 import re
 import subprocess
 import sys
-from base64 import b32decode
+import urllib.parse
 
 import onetimepass
 
 
+DIGITS_DEFAULT = 6
+
+
+class BackendError(Exception):
+    backend_name = '<none>'
+
+    def __init__(self, msg):
+        self.msg = msg
+
+class PassBackendError(BackendError):
+    backend_name = 'pass'
+
+
+def _read_backend_error(err):
```

Comment:
Maybe it makes more sense for `PassBackendError` to receive this `err, bytes` as params and move this function into its constructor?

As far as I can tell, this function is pretty much tied to invocations of `PassBackendError`.

Reply:
Right, I've moved it there!